apps/web-giddh/src/app/theme/sales-ng-virtual-select/sh-select.component.scss

```diff
@@ -104,29 +104,29 @@ i.close.icon.clear:hover::after {
 }
 
 .selected-val {
-    border: 1px solid $light-gray !important;
```

Comment:
@aditi-tiwari17 @shalinee-walkover Please check this code revert

Reply:
checked, its fine